src/Components/Home.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './styles/Home.css';
```

Comment:
Hello @mrparkersson , you have a good job on this milestone and your project code looks fine with the routes well setup. However, when I start the server, the project compiles with some errors as seen below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/153962729-c04ff64a-c154-4342-819a-ce46f8c2e64e.png)